profile/README.md

```diff
@@ -1,65 +1,66 @@
 # Lombiq Technologies 
 
 
-[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
-Our main area of expertise is the Orchard Core content management system. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
-
-## Products:
-
-* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips (FPGA). You can still update your software, but it will be a whole lot faster and more power-efficient.
-
- <img src="https://user-images.githubusercontent.com/68704777/173390183-8c2e83ae-9ce3-47c4-a62e-4c58d83648e0.png" width="1000" height="250">
-
 
-* [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the [Orchard Core content management system](https://www.orchardcore.net/). We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS, [DotNest](https://dotnest.com).
 
 
-<img src="https://user-images.githubusercontent.com/68704777/173390495-9c1fa7a2-084c-4a9b-bc6b-c59e183fb784.png" width="400" height="400">
- 
 ## Our services:
 
-### Software development:
- 
+### Software development: 
 Our team's passion is developing software with care for clients who need help with their most extraordinary projects in the following frameworks:
 
-- Orchard Core.
-- Orchard CMS.
-- ASP.NET Core.
-- ASP.NET.
+- Orchard Core
+- Orchard CMS
+- ASP.NET Core
+- ASP.NET
 
 ### Consulting:
-
 We offer Orchard consulting in both Orchard editions, Orchard Core and Orchard CMS, including website and application development, and code review.
 
 ### Training:
-
 Our team of dedicated experts is here to help groups and individuals who want to learn about Orchard or simply broaden their coding knowledge. Our [Orchard Dojo](https://orcharddojo.net/) website contains several training sessions to help beginners become experts.
 
+![Orchard Dojo](https://user-images.githubusercontent.com/68704777/174299406-a7ea1222-a1a8-457e-b68f-e3570bf2d5b6.png)
```

Comment:
Let's leave out these logos, actually, because they look bad in dark mode:

![image](https://user-images.githubusercontent.com/1976647/174353035-3ff5bbf5-2b31-4ac2-bb6d-1ab21a6173cb.png)